README.md

```diff
@@ -13,57 +13,39 @@ It is also possible to use Reshade Fx shaders.
 ## Disclaimer
 This is one of my first projects ever, so expect it to have bugs. Use it at your own risk.
 
-## Install
-**There are binaries attached to each release, so manually building from source is not required**
+## Building from Source
 
-Once you have the binaries, either by downloading and unpacking them or by following the build instructions below, execute:
-```
-make install
-```
-**Note: do not run this with `sudo`. vkBasalt gets installed for the current user only.**
-
-## Build Dependencies
-
-### Prerequisites
+### Dependencies
 Before building, you will need:
 - GCC >= 9
 - X11 development files
 - glslang
-- spirv-opt
 
-See below on how to install them.
+### Building
+
+**Make sure that the resulting install directory for the .so is found by the linker, otherwise the loader will not find it.** In general, prefer using distro provided packages.
 
-#### Arch-based distributions
-For Arch-based distributions, execute:
-```
-sudo pacman -Syu glslang vulkan-tools lib32-libx11 libx11
-```
-#### Debian/Ubuntu-based distributions
-For newer Debian/Ubuntu-based distributions, execute:
-```
-sudo apt install build-essential gcc-multilib libx11-dev libx11-dev:i386 glslang-tools spirv-tools
 ```
-#### Fedora
-For Fedora, execute:
+git clone https://github.com/DadSchoorse/vkBasalt.git --recurse-submodules
+cd vkBasalt
 ```
-sudo dnf install vulkan-tools glslang libX11-devel glibc-devel.i686 libstdc++-devel.i686 spirv-tools libX11-devel.i686
+
+#### 64bit
+
 ```
-#### Gentoo-based distributions
-For Gentoo-based distributions, execute:
+meson --buildtype=release build.64
+ninja -C build.64 install
 ```
-sudo emerge -v dev-util/glslang dev-util/vulkan-tools dev-util/spirv-tools
+#### 32bit
+
+Make sure that `PKG_CONFIG_PATH=/usr/lib32/pkgconfig` and `--libdir=lib32` are correct for your distro and change them if needed. 
```

Comment:
This isn't true for every distro afaik. While the FHS promotes using `lib32` and `lib64`, Debian based distro use a more general declaration where to put the libs (it includes for example the arch and the kernel: https://wiki.debian.org/Multiarch/TheCaseForMultiarch).

Reply:
That's why this note exists :frog:
I'm not going to document every distro's way to handle this. If you want I can change it to a harsher note.

Reply:
I guess it's fine, I just would make a bit clearer to reduce bug reports that only read the first sentence. Something like
```
Make sure to set the correct path for your distro, on FHS 3.0 that is ...
```

Reply:
I think I will leave it as it is.